Source/Jenkins/jenkins-api.env

```diff
@@ -19,9 +17,52 @@ _t4dDebugInfoStart "_t4dSrcJenkinsApiGetPage" "$#" "$@"
 ###### DOCUMENTATION BLOC #####
 #EOF
 ########## CODE BLOC ##########
+    local _t4dErrorCode=0
```

Comment:
catching error code will allow us to return a better errorCode to upper functions